inverseCase/inverseCase.js

```diff
@@ -0,0 +1,12 @@
+var toUpper = require('../toUpper/toUpper');
+var toLower = require('../toLower/toLower');
+
+String.prototype.inverseCase = function() {
+  var newString = '', regex = /[a-z]/;
+  for (var i = 0; i < this.length; i++) {
+    regex.test(this[i]) ? newString += this[i].toUpper() : newString += this[i].toLower();
```

Comment:
Add a comment here to explain what the code base is doing, or maybe actually use an if statement to carry out this operation